### DISCUSSION TITLE
LXC for SearxNG search engine

### DISCUSSION THREAD

**Discussion by jotenakis2** [community-scripts/ProxmoxVE#27] [Request script]
This script is working.
Create a debian 12 LXC then run the script and tada you have searxng self hosted.
https://github.com/tteck/Proxmox/discussions/318#discussioncomment-11128753

**Comment by MickLesk** [comment]
Please use next time "Request Script", that makes everything smarter and easier to filter.

By the way, I get palpitations when I see the script. I have to see if it can be bent to conform to the project. 😅

**Reply by Vakrehus** [reply]
That would be very nice. I just put this together from the website for my own purposes.

**Comment by Mati-l33t** [comment]
This script is working. Create a debian 12 LXC then run the script and tada you have searxng self hosted. [tteck#318 (reply in thread)](https://github.com/tteck/Proxmox/discussions/318#discussioncomment-11128753)

Not really.

```shell
Debian GNU/Linux 12 debian tty1

debian login: root
Password: 
Debian LXC provided by https://helper-scripts.com/

root@debian:~# bash -c “$(wget -qLO - https://raw.githubusercontent.com/Vakrehus/pr0xm0x/main/searxng-lxc-install.sh)”
#: line 1: “#!/bin/bash: No such file or directory
root@debian:~# 
```

**Reply by Vakrehus** [reply]
It worked with tteck debian 12. The issue here is with the quotation marks used.

`bash -c "$(wget -qLO - https://raw.githubusercontent.com/Vakrehus/pr0xm0x/main/searxng-lxc-install.sh)"`

**Comment by havardthom** [comment]
Proper script request for tracking here https://github.com/community-scripts/ProxmoxVE/discussions/122. Closing this.
